main.sh

```diff
@@ -138,28 +136,17 @@ create_pool(){
 #Start daos servers
 start_server(){
     echo -e "\nCMD: Starting server...\n"
-    rm -r server_output.log
-    cmd="orterun  --np $DAOS_SERVERS --hostfile Log/$SLURM_JOB_ID/daos_server_hostlist -x CPATH -x PATH -x LD_LIBRARY_PATH 
-    --enable-recovery daos_server start -i -o $DAOS_SERVER_YAML  
-    --recreate-superblocks 2>&1 | tee server_output.log"
+    cmd="clush --hostfile Log/$SLURM_JOB_ID/daos_server_hostlist \"
+    export PATH=$PATH; export LD_LIBRARY_PATH=$LD_LIBRARY_PATH; export CPATH=$CPATH;
+    export DAOS_DISABLE_REQ_FWD=1;
+    daos_server start -i -o $DAOS_SERVER_YAML  
+    --recreate-superblocks
```

Comment:
I think we should stop using '--recreate-superblocks' as recommended by the control plane team in recent times.

Reply:
We don't have root access to run dmg storage format on frontera.